external/curl/Makefile

```diff
@@ -63,6 +63,7 @@ $(COBJS): curl_%$(OBJEXT): %.c
 	$(call COMPILE, $<, $@)
 
 .built: $(OBJS)
+	cp curl-certificates.crt $(TOPDIR)/../tools/fs/contents
```

Comment:
Please add removing this file from contents folder when clean or distclean is called.

Reply:
Okey. Changes done.